modules/ROOT/pages/anypoint-mq/2.x/anypoint-mq-connector-reference.adoc

```diff
@@ -0,0 +1,474 @@
+= Anypoint MQ Connector Reference - 2.x
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint MQ is a multi-tenant, cloud messaging service that enables customers to perform advanced asynchronous messaging scenarios between their applications.
+
+*Anypoint MQ Connector version 2.0.5*
+
+November 2, 2018
+
+
+== Configurations
+---
+[[default-subscriber]]
+=== Default Subscriber
+
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | |x
+| Connection a| <<default-subscriber_connection, Connection>>
+ | The connection types that can be provided to this configuration. | |x
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime actually purges the instances as needed. |  |
+| Acknowledgement Mode a| Enumeration, <<SubscriberAckMode>>:
+
+** IMMEDIATE
+** AUTO
+** MANUAL |  The Acknowledgment mode to use for the messages retrieved by the subscribers. This affects only the Subscriber source and is unrelated to the consume operation. Refer to the <<SubscriberAckMode>> for more details on each strategy. |  AUTO |
+| Acknowledgement Timeout a| Number |  States the duration that a message is held by a broker waiting for an Acknowledgment or Not Acknowledgment. If neither an ACK or a NACK occurs before the timeout is exceeded, then the message is automatically returned to the destination by the broker, making it again available to any subscriber for redelivery. By default, this timeout is decided by the Subscriber depending on which strategy is used. When doing polling, then the ACK timeout is the one configured in the queue. When using the Prefetch strategy, messages are kept in a local buffer until they are dispatched to the flow. You can update the time to live (TTL) to avoid having messages be evicted. By default, the messages are fetched with a TTL of at least 2 minutes. |  0 |
+| Polling Time a| Number |  When prefetch is disabled, the polling time defines how often the subscriber should check for new messages. By default, this checks for messages every one second (only if the prefetch is disabled). The value of this parameter is defined in milliseconds. |  1000 |
+| Max Redelivery a| Number |  Number of redeliveries to accept for any given message. When the message redelivery count is higher than this threshold, then the message is ignored by the subscriber and not dispatched to the flow. For the message to be removed from the queue, the queue should be configured with a dead letter queue from the admin console. |  -1 |
```

Comment:
"the queue should be configured with a...queue"? Reword for clarity.
When possible, write in active voice: "For...queue, configure the queue...."